src/ng/parse.js

```diff
@@ -839,13 +867,16 @@ ASTCompiler.prototype = {
 
   watchFns: function() {
     var result = [];
-    var fns = this.state.inputs;
+    var inputs = this.state.inputs;
     var self = this;
-    forEach(fns, function(name) {
-      result.push('var ' + name + '=' + self.generateFunction(name, 's'));
+    forEach(inputs, function(input) {
+      result.push('var ' + input.name + '=' + self.generateFunction(input.name, 's'));
```

Comment:
is there any performance/code-size gain in setting and using `var name = input.name;` here?
Or does the minification process deal with that automatically?